pyproject.toml

```diff
@@ -1,13 +1,13 @@
 [tool.poetry]
-name = "poetry-bundle-plugin"
+name = "poetry-plugin-bundle"
```

Comment:
might also need a dev dependency on poetry@master and poetry >= 1.2.0.b1.dev0

Reply:
done